File: hw/otavalo/u1/HW08SaveInfomationInCsvAndTxt/CSV in Python/Chicken_model.py
class Chicken:
    id_counter = 1

    def __init__(self, name, color, age, is_molting):
        self.id = Chicken.id_counter
        Chicken.id_counter += 1
        
        self.name = name
        self.color = color
        self.age = age
        self.is_molting = is_molting

    def __str__(self):
        molting_status = 'True' if self.is_molting else 'False'
        return f"Chicken{{ id -> {self.id}, name -> {self.name}, color -> {self.color}, age -> {self.age}, isMolting -> {molting_status}}}" 

File: hw/medina/u1/hw06ChickenFarmSimulator/chicken-JavaScript/Chicken_model.js
class Chicken {
   /**
     @param {number} id 
     @param {string} name
      @param {string} color 
     @param {number} age 
     @param {boolean} isMolting 
     */
    constructor(id, name, color, age, isMolting) {
          this.id = id;
          this.name = name;
          this.color = color;
          this.age = age;
          this.isMolting = isMolting;
    }

     toString() {
          return `Chicken: id=${this.id}, name=${this.name}, color=${this.color}, age=${this.age}, isMolting=${this.isMolting}`;
    }
}
module.exports = Chicken;